docs/user/config_file.md

```diff
@@ -0,0 +1,43 @@
+# Introducing
+
+Daemon config file is a entry for user to set flags for pouch daemon. Pouch
+support two ways for users to pass flags to daemon, one is run pouch daemon
+directly with flags specified, like: `pouchd -c /var/run/containerd.sock`, the
+other one is using daemon config file, and of course, you can use them simultaneously.
+
+## Support daemon flags
+
+We list some flags pouch daemon supports, for the detail flags explanations, you
+can find in [pouchd flags](https://github.com/alibaba/pouch/blob/master/docs/commandline/pouch.md).
+
+| Flag                  | Description                             |
+|-----------------------|-----------------------------------------|
+| `-c`, `--containerd`  | where does containerd listened on. |
+| `-l`, `--listen`      | which address to listen on.            |
+
+## Configuring pouch daemon config file
+
+We recommend users to set daemon flag through daemon config file, the default
+path to config file is `/etc/pouch/config.json`, you can change it by set
+value of `--config-file`.
+
+### Note
+
+* The same flag(exclude slice or array type) can not be set through command
```

Comment:
maybe need a blank line here

Reply:
no, if add a blank line here, it will cause another error,  which means just allow one blank line

Reply:
The rules confuse me very much !!!

Reply:
Hi @Ace-Tang , the rule is here. https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md032---lists-should-be-surrounded-by-blank-lines

I think you can add the indent in the newline. Please have a try. haha

Reply:
As the text say
```
Some text

* Some
* List
```

I must add a text before `*some`, but if I do not want to add text, I just want `* some` follow `## title`, how do I need to do.